packages/core/src/Table/Table.stories.js

```diff
@@ -17,190 +22,458 @@ Should be used with multiple Table-related child components - see the table and
-                <TableCell>01/03/2006</TableCell>
-                <TableCell>28</TableCell>
-                <TableCell>Dalakuru CHP</TableCell>
-                <TableCell>Anatoliy Shcherbatykh</TableCell>
-                <TableCell>Incomplete</TableCell>
+                <TableCell tag="th" scope="row" rowSpan="2">
+                    Type
+                </TableCell>
+                <TableCell tag="th" scope="row">
+                    Teddy Bears
+                </TableCell>
+                <TableCell>50,000</TableCell>
+                <TableCell>30,000</TableCell>
+                <TableCell>100,000</TableCell>
+                <TableCell>80,000</TableCell>
             </TableRow>
-        </TableBody>
-        <TableFoot>
             <TableRow>
-                <TableCell colSpan="8">
-                    <TableFooterButton />
+                <TableCell tag="th" scope="row">
+                    Board Games
                 </TableCell>
+                <TableCell>10,000</TableCell>
+                <TableCell>5,000</TableCell>
+                <TableCell>12,000</TableCell>
+                <TableCell>9,000</TableCell>
             </TableRow>
-        </TableFoot>
+        </TableBody>
     </Table>
 )
+export const IrregularHeaders = IrregularHeadersTemplate.bind({})
+IrregularHeaders.args = {}
+
+const ToolbarsTemplate = args => (
+    <>
+        <TableToolbar>
+            <p>Content</p>
+        </TableToolbar>
+        <Table {...args}>
+            <TableHead>
+                <TableRow>
+                    <TableCellHead>First name</TableCellHead>
+                    <TableCellHead>Last name</TableCellHead>
+                    <TableCellHead>Incident date</TableCellHead>
+                    <TableCellHead>Last updated</TableCellHead>
+                </TableRow>
+            </TableHead>
+            <TableBody>
+                <TableRow>
+                    <TableCell>Onyekachukwu</TableCell>
+                    <TableCell>Kariuki</TableCell>
+                    <TableCell>02/06/2007</TableCell>
+                    <TableCell>05/25/1972</TableCell>
+                </TableRow>
+                <TableRow>
+                    <TableCell>Kwasi</TableCell>
+                    <TableCell>Okafor</TableCell>
+                    <TableCell>08/11/2010</TableCell>
+                    <TableCell>02/26/1991</TableCell>
+                </TableRow>
+                <TableRow>
+                    <TableCell>Siyabonga</TableCell>
+                    <TableCell>Abiodun</TableCell>
+                    <TableCell>07/21/1981</TableCell>
+                    <TableCell>02/06/2007</TableCell>
+                </TableRow>
+            </TableBody>
+        </Table>
+        <TableToolbar position="bottom">
+            <p>Content (bottom)</p>
+        </TableToolbar>
+    </>
+)
+export const Toolbars = ToolbarsTemplate.bind({})
+Toolbars.args = {}
+
+const ExpandableContentTemplate = args => {
```

Comment:
I think this story can be removed as the more interactive `Expandable Table Row` story below showcases the feature better.